storage/s3/storage.go

```diff
@@ -18,3 +25,19 @@ func SetToContext(c *gin.Context, s3client s3iface.S3API) {
 func GetFromContext(c context.Context) s3iface.S3API {
 	return c.Value(key).(s3iface.S3API)
 }
+
+// CreateTemplate uploads html file to s3.
+func CreateTemplate(c context.Context, tmplInput *entities.Template) error {
+
+	input := &s3.PutObjectInput{
+		Bucket: aws.String(os.Getenv("AWS_S3_BUCKET")),
+		Key:    aws.String(fmt.Sprintf("/PATH_TO_FILE/%d/%s", tmplInput.UserID, tmplInput.Name)),
+		Body:   bytes.NewReader([]byte(tmplInput.HTMLPart)),
+	}
+
+	_, err := GetFromContext(c).PutObject(input)
```

Comment:
Instead of using `GetFromContext(c)` we should have a wrapper for this, like we do with the other methods. For example: https://github.com/mailbadger/app/blob/master/storage/storage.go#L120